languages/python/exercises/concept/basics/.docs/instructions.md

```diff
@@ -0,0 +1,51 @@
+You're going to write some code to help you cook a brilliant lasagna from your favorite cookbook.
+
+You have five tasks, all related to cooking your recipe.
+
+## 1. Define expected bake time in minutes
+
+Define an `EXPECTED_BAKE_TIME` constant that returns how many minutes the lasagna should bake in the oven. According to your cookbook, the Lasagna should be in the oven for 40 minutes:
+
+```python
+>>> lasagna.EXPECTED_BAKE_TIME
+40
+```
+
+## 2. Calculate remaining bake time in minutes
+
+Implement the `bake_time_remaining()` function that takes the actual minutes the lasagna has been in the oven as an argument and returns how many minutes the lasagna still needs to bake based on the `EXPECTED_BAKE_TIME`.
+
+```python
+>>> bake_time_remaining(30)
+10
+```
+
+## 3. Calculate preparation time in minutes
+
+Implement the `preparation_time_in_minutes()` function that takes the number of layers you want to add to the lasagna as an argument and returns how many minutes you would spend making them. Assume each layer takes 2 minutes to prepare.
+
+```python
+>>> preparation_time_in_minutes(2)
+4
+```
+
+## 4. Calculate total elapsed cooking time (prep + bake) in minutes
+
+Implement the `elapsed_time_in_minutes()` function that has two parameters: `number_of_layers` (_the number of layers added to the lasagna_) and `elapsed_bake_time` (_the number of minutes the lasagna has been baking in the oven_). This function should return the total number of minutes you've been cooking, or the sum of your preparation time and the time the lasagna has already spent baking in the oven.
+
+```python
+>>> elapsed_time_in_minutes(3, 20)
+26
+```
+
+## 5. Add docstrings and comments to your work
```

Comment:
Hard agree.  :-)  I was debating it when I wrote it because adding "notes" might lead someone to put in things about what page the recipe was on, etc....but I think we have enough hints and links to let the student know it's for their code -- and it will read better.